### PR TITLE
Update ResourceCommand.php

### PR DESCRIPTION
**Pull request by Geoffry304**
Add the force case insensitive to make the search case insensitive for searching in json fields, for example translation fields.

Copied the code from the Filament Resource and added the current resource to the function.

### CODE REVIEW

**Comment by pxlrbt** [comment]
Sorry, closing this as the PR is a bit old and I don't fully understand all the changes. Happy to reopen if you are still interested.

**Comment by Geoffry304** [comment]
Hello

If you have a field that you store in the database as json and you try to search on this when resource 'isGlobalSearchForcedCaseInsensitive' is set to true then the record will not be displayed when you go to search CaseInsensitive.

```
{"de": "GROEP DE", "en": "GROEP EN", "fr": "GROEP FR", "nl": "Groep NL"}
```

This is what you get when using the code in my pull request:
image
image

And this is what you get when using your code:
image
image


I hope it is more clear now :)

File: src/Commands/ResourceCommand.php [hunk 1]
```diff
@@ -118,30 +121,33 @@ public function searchRecord($query): EloquentCollection|Collection|array
...
-            'pgsql' => 'ilike',
-            default => 'like',
-        };
+        $isForcedCaseInsensitive = $resource::isGlobalSearchForcedCaseInsensitive();
 
+        /** @var Connection $databaseConnection */
+        $databaseConnection = $query->getConnection();
+        if ($isForcedCaseInsensitive){
+            $searchQuery = strtolower($searchQuery); 
+        }
```
Comment (pxlrbt) [inline src/Commands/ResourceCommand.php:134]: Why do we need to convert to lowercase? I compared to Filament's code and it's not there

Comment (Geoffry304) [reply src/Commands/ResourceCommand.php:134]: Otherwise it wont show when searching for 'Groep'

**Comment by pxlrbt** [comment]
I had to create a new PR since I couldn't push changes to your branch. #50

File: src/Commands/ResourceCommand.php (hunk 1 above)
Comment (Geoffry304) [reply src/Commands/ResourceCommand.php:134]: The ```generate_search_column_expression``` is used to convert the field in the database to lowercase, so thats why I convert the ```$searchQuery``` to lowercase.

**Pull request merged**